File: app-ui/MediWeckerUI/Application/Features/Planning/TimepointType.cs
using System.Runtime.Serialization;

namespace MediWeckerUI.Application.Features.Planning;

public enum TimepointType
{
    [EnumMember(Value = "0")]
    AbsoluteTime,
    
    [EnumMember(Value = "1")]
    Morning,
    
    [EnumMember(Value = "2")]
    Midday,
    
    [EnumMember(Value = "3")]
    Evening,
    
    [EnumMember(Value = "4")]
    Night
}

File: app/src/main/java/de/htwBerlin/ai/mediAlarm/data/rhythm/TimepointType.kt
package de.htwBerlin.ai.mediAlarm.data.rhythm

import com.google.gson.annotations.SerializedName

enum class TimepointType {
    @SerializedName("0")
    AbsoluteTime,
    @SerializedName("1")
    Morning,
    @SerializedName("2")
    Midday,
    @SerializedName("3")
    Evening,
    @SerializedName("4")
    Night
}
